instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from './CommentForm';
```

Comment:
Pay attention to your file paths - CommentForm is not in the same folder as CommentSection. You can go up a folder by using `../`